res/values/strings.xml

```diff
@@ -617,7 +617,7 @@
     <string name="pref_watch_sent_folder">Watch Sent Folder</string>
     <string name="pref_send_copy_to_self">Send Copy to Self</string>
     <string name="pref_send_copy_to_self_explain">Required when using this account on multiple devices.</string>
-    <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat Folder</string>
+    <string name="pref_auto_folder_moves">Move automatically to DeltaChat Folder</string>
```

Comment:
the old one looks a bit better to me in the context of the "IMAP Folder Handling" - or is the old one just wrong english?

this is not a blocker, however.

Reply:
The old one wasn't wrong but a bit more awkward and opaque, there's no ambiguity about the second one.